packages/fx-core/src/plugins/resource/simpleauth/utils/common.ts

```diff
@@ -37,13 +37,10 @@ export class Utils {
 
         const version = await fs.readFile(versionFilePath, "utf-8");
         const fileName = Constants.SimpleAuthZipName(version);
-        const blobUrlWithCredential = process.env.SIMPLE_AUTH_URL as string;
 
         try {
-            const blobClient = new BlobServiceClient(blobUrlWithCredential)
-                .getContainerClient("release")
-                .getBlobClient(fileName);
-            await blobClient.downloadToFile(filePath);
+            const isPrelease = true;
+            await downloadRelease(Constants.GithubUserName, Constants.GithubRepoName, filePath, ()=>{ return isPrelease}, (asset)=>{ return asset.name.includes(Constants.SimpleAuthName)});
```

Comment:
is downloadRelease promise? If not we may not use await. The code is weird since it use await and also callback. Please check the api